bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
@jsimsa, bin/tachyon-mount.sh sets $bin environment variable under function init_env when *mount_local* or *workers* 2nd parameter is passed , my thought was that you want to make a better variable assignment removing the two dots